test/utils/url.spec.ts

```diff
@@ -80,11 +80,15 @@ describe("Url utils tests", () => {
 
   describe("isRemoveJourney tests", () => {
 
-    test("returns true if app data not present in session and query param journey=remove", () => {
+    test.each([
+      ['journey=remove', 'remove'],
+      ['journey=remove&journey=remove', 'remove,remove'],
+      ['journey=remove&journey=update', 'remove,update']
```

Comment:
This test case raises the question what _should_ the `isRemoveJourney()` function return in this scenario?

Probably not **true**, as if two different values are set we can't be sure what journey the user is on. I'd have thought it should throw an error.

Reply:
It could do, i also thought in that case isRemoveJourney and isUpdateJourney functions could both return true. Perhaps the error is the better option.

Reply:
I think an error is better. You can't be on both journeys at the same time.